acbuild/acbuild.go

```diff
@@ -249,7 +249,10 @@ func main() {
 	// Make help just show the usage
 	cmdAcbuild.SetHelpTemplate(`{{.UsageString}}`)
 
-	cmdAcbuild.Execute()
+	err := cmdAcbuild.Execute()
+	if cmdExitCode == 0 && err != nil {
+		os.Exit(3)
```

Comment:
Because I know I used 1 in some places, and couldn't remember if I used 2. Why 63 or 105?

> On Feb 19, 2016, at 07:59, Jonathan Boulle <notifications@github.com> wrote:
> 
> In acbuild/acbuild.go:
> 
> > @@ -249,7 +249,10 @@ func main() {
> >  	// Make help just show the usage
> >  	cmdAcbuild.SetHelpTemplate(`{{.UsageString}}`)
> >  
> > -	cmdAcbuild.Execute()
> > +	err := cmdAcbuild.Execute()
> > +	if cmdExitCode == 0 && err != nil {
> > +		os.Exit(3)
> why 3 and not 63 or 105?
> 
> ―
> Reply to this email directly or view it on GitHub.
>